utils/publish-legacy.sh

```diff
@@ -25,4 +33,6 @@ for tag in $tags; do
 
   echo "Running \`npm publish $npm_tag\` in \"$(pwd)\""
   npm publish $npm_tag
+  echo "Running \`npm deprecate $old_name@$version\` in favor of $new_name"
+  npm deprecate "$old_name@$version" "\"$old_name\" is deprecated and will stop receiving updates January 2021. Please use \"$new_name\" instead."
```

Comment:
```suggestion
  npm deprecate "$old_name@$version" "\"$old_name\" is deprecated and will stop receiving updates after January 2021. Please use \"$new_name\" instead."
```

Reply:
It will stop receiving updates during Jan, not after 🤔 

The last publish will be Dec 2020 and we stop Jan 2021